### PR TITLE
(redoing)thirdperson fix,,antiaim ,an autostrafe, doorspam

### PR DESCRIPTION
**Pull request by notgoodusename**
redoing this

### CODE REVIEW

**Comment by ZPDMiller** [comment]
Thank you

**Comment by littlehurensohn** [comment]
Still flicking in the gotv, and when somebody spectating you.. ;)

**Comment by notgoodusename** [comment]
> Still flicking in the gotv, and when somebody spectating you.. ;)

i know i didnt test it its really hard to make a proper desync

**Comment by Oly-UK** [comment]
yo wtf is QAngle? do i need some kind of vector file?

**Comment by notgoodusename** [comment]
sorry forgot to put qangle definition

**Comment by danielkrupinski** [comment]
Use Vector instead of QAngle.

**Comment by Novalaki** [comment]
I seen you post about the auto strafer not working on UC. guess you finnaly figured it out you didnt have a movement fix LOL

**Comment by notgoodusename** [comment]
Yeah it was me figured that i needed to fixmovement for aa not for createmove thundef4ll helped me a little by private

**Comment by Novalaki** [comment]
Well thats good at least. guys on UC can be dicks when they see that new people want help

**Comment by Novalaki** [comment]
Thanks for the contribution by the way! i find this useful.

**Comment by notgoodusename** [comment]
Thanks! rn im not home but i will post qanlge and other functions

**Comment by Oly-UK** [comment]
thanks! 
also your anti aim seems to be not creating a fake desync at all as when i record a demo it is from the perspective of the anti aim

**Comment by notgoodusename** [comment]
im aware of this im trying to make it work

**Comment by thomas0aotian** [comment]
And for some reason I can't compile it. It says the problem is in anti aim.h 
Can u check again.tks

**Comment by thomas0aotian** [comment]
yeah the legit aa wasn't desyncing

**Comment by thomas0aotian** [comment]
wait hold up it needs to cooperate with the choke package

**Comment by thomas0aotian** [comment]
still

File: Osiris/Hacks/Misc.h [hunk 1]
```diff
@@ -12,6 +12,7 @@
 #include "../SDK/GlobalVars.h"
 
 namespace Misc {
+    void slowawalk(UserCmd* cmd) noexcept;
```
[the remark this answers is not in the document]
Comment (notgoodusename) [reply Osiris/Hacks/Misc.h]: Em it should be like that

[the remark this answers is not in the document]
Comment (ZPDMiller) [reply Osiris/Hacks/Misc.h]: What do you mean? It's "slowwalk" in the other files

[the remark this answers is not in the document]
Comment (notgoodusename) [reply Osiris/Hacks/Misc.h]: idk he thought it was misc::slowwalk instead of slowwalk

[the remark this answers is not in the document]
Comment (ZPDMiller) [reply Osiris/Hacks/Misc.h]: He's saying that the typo has an "a" in between the two "w"s, like "slowawalk", not "slowwalk"

[the remark this answers is not in the document]
Comment (notgoodusename) [reply Osiris/Hacks/Misc.h]: yeah but still he didnt fix it lol

**Comment by Oly-UK** [comment]
Anyone know why im getting the following errors?
syntax error: indentifier 'QAngle'
missing type specifier - int assumed. Note: C++ does not support default-int
syntax error: missing '.' before '&'

**Comment by notgoodusename** [comment]
Look at vector.h

**Comment by Oly-UK** [comment]
i am confusion

**Comment by Oly-UK** [comment]
nevermind sorted it, thanks :)

**Comment by thomas0aotian** [comment]
@Oly-UK bruh how did u fix it i got the same error codes too

**Comment by Oly-UK** [comment]
literally just include vector in antiaim.h

**Comment by thomas0aotian** [comment]
@Oly-UK lol tf ,alright ima do it later.tks

File: Osiris/Hacks/Misc.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (thomas0aotian) [reply Osiris/Hacks/Misc.h]: yeah i thought ionly can comment on it

**Comment by Novalaki** [comment]
Sorry for the stupid question. but whats the max slow walk amount

**Comment by thomas0aotian** [comment]
@Novalaki max slowwalk speed?

**Comment by Novalaki** [comment]
@thomas0aotian yeah

**Comment by Oly-UK** [comment]
i dont know if i did something wrong but the anti aim only works for a few seconds then the legs rotate back to normal

**Comment by notgoodusename** [comment]
legs shouldnt rotate at all when desyncing otherwise it will fuck up the process

**Comment by Oly-UK** [comment]
oh ok i think i know why that's happening, i'll need some form of desync chams or this will fry my brain xD

**Comment by notgoodusename** [comment]
check velocity while desyncing if you move 2 or 1 unit it should be working

**Comment by Oly-UK** [comment]
https://prnt.sc/ptdz0e
yeah it's working good job

**Comment by littlehurensohn** [comment]
Somebody who tested it, still flicking or no?

**Comment by notgoodusename** [comment]
i tested it on casual not kicked once, i recommend choking 2 packets instead of 1 works better

**Comment by thomas0aotian** [comment]
Ima test it out later

**Comment by thomas0aotian** [comment]
And u can not turn on desync in casual,it will get u vacae

**Comment by thomas0aotian** [comment]
@notgoodusename if the desync doesn't auto cooperate with choke package, then idt it will work. 12 is the largest amount of package that u can choke and doesn't effect ur normal gameplay.

**Comment by notgoodusename** [comment]
ok

**Comment by HyeN2018** [comment]
I donwloaded the version that u edited and it gives some erorrs in VS2017 about anti aim

**Comment by notgoodusename** [comment]
tell the errors you get

**Comment by HyeN2018** [comment]
> tell the errors you get

https://prnt.sc/ptwzjf

**Comment by notgoodusename** [comment]
That should fix it

**Comment by Oly-UK** [comment]
oml
and i thought i was bad at fixing things

**Comment by notgoodusename** [comment]
al least he told me xd

**Comment by HyeN2018** [comment]
Still not working : https://imgur.com/a/U0Mxv0J

**Comment by notgoodusename** [comment]
will look later

**Comment by Oly-UK** [comment]
@HyeN2018 #include "Vector.h" in antiaim.h

**Comment by rosesware** [comment]
@Oly-UK not working.

**Comment by rosesware** [comment]
@HyeN2018 just put `#include "../SDK/Vector.h"` on `AntiAim.h`.

**Comment by littlehurensohn** [comment]
I was testing this in matchmaking, it was works but.. 
Its still flicking, when somebody spectating you, or when u check the matchmaking demo. 
For now I recommend dont use it in matchmaking.

**Comment by notgoodusename** [comment]
The code is correct, Chocked packets cant be seen by other players so what are you saying doesnt make sense

**Comment by notgoodusename** [comment]
> Btw: When u have toggled on Legit AntiAim, and somebody kill you -> game crash.

Do you have that aa nade fix? or something else?

**Comment by littlehurensohn** [comment]
> > Btw: When u have toggled on Legit AntiAim, and somebody kill you -> game crash.
> 
> Do you have that aa nade fix? or something else?

Yes

**Comment by notgoodusename** [comment]
> > > Btw: When u have toggled on Legit AntiAim, and somebody kill you -> game crash.
> > 
> > 
> > Do you have that aa nade fix? or something else?
> 
> Yes

aight then thats the problem

**Comment by littlehurensohn** [comment]
> > > > Btw: When u have toggled on Legit AntiAim, and somebody kill you -> game crash.
> > > 
> > > 
> > > Do you have that aa nade fix? or something else?
> > 
> > 
> > Yes
> 
> aight then thats the problem

Just when i remove the aa nade fix it be works, right. 
But, what about the nade check? Cause when actually u be want throw the nade, nade going out.. 
Thats a make little problem.

**Comment by notgoodusename** [comment]
> > > > > Btw: When u have toggled on Legit AntiAim, and somebody kill you -> game crash.
> > > > 
> > > > 
> > > > Do you have that aa nade fix? or something else?
> > > 
> > > 
> > > Yes
> > 
> > 
> > aight then thats the problem
> 
> Just when i remove the aa nade fix it be works, right.
> But, what about the nade check? Cause when actually u be want throw the nade, nade going out..
> Thats a make little problem.

will try to fix it, btw what do you mean by flicking? do you do record "name" or tv_record "name" record is client side and will show choked and unchoked packets, while tv record will show what the server sees

**Comment by littlehurensohn** [comment]
Bruh, I was testing it in matchmaking. 
Just after the matchmaking I check the demo, and my teammates was telling me my screen flicking so hard to the right side.

**Comment by notgoodusename** [comment]
> Bruh, I was testing it in matchmaking.
> Just after the matchmaking I check the demo, and my teammates was telling me my screen flicking so hard to the right side.

weird that shouldnt happen

**Comment by littlehurensohn** [comment]
I was testing antiaim again, its not flicks. 
But for me it works only when i choke 14 packets. 
And still the problem with the nade, but i glad u know how to fix it :P 
And btw.. Antiaim is works, but there is still too big chance to hit head, sometimes they can normally hit my head.

**Comment by littlehurensohn** [comment]
But nice work ( : Mission Passed -> Respect+!

**Comment by notgoodusename** [comment]
> And still the problem with the nade, but i glad u know how to fix it :P
trying
> And btw.. Antiaim is works, but there is still too big chance to hit head, sometimes they can normally hit my head.

thats because im using sidemoves, not an lby breaker,  im trying to implement it for better results while desyncing

**Comment by xsincbt** [comment]
how do i use the block bot

**Comment by Oly-UK** [comment]
YOU ADDED A BLOCKBOT? YOU'RE A FUCKING GOD

**Comment by Oly-UK** [comment]
@notgoodusename @littlehurensohn is right though, it does flicker with choke packets

**Comment by littlehurensohn** [comment]
> @notgoodusename @littlehurensohn is right though, it does flicker with choke packets

he fix it.

**Comment by thomas0aotian** [comment]
Bruh u wanna puy block bot on key? We don't want it to be always on when toggled

**Comment by notgoodusename** [comment]
will do later cant right now

**Comment by thomas0aotian** [comment]
Yeah bruh still a gj

**Comment by giangero1** [comment]
image
got ton of error on misc.cpp can someone post their whole misc.cpp file cause i copy pasted just like you said :( @notgoodusename

**Comment by notgoodusename** [comment]
> image
> got ton of error on misc.cpp can someone post their whole misc.cpp file cause i copy pasted just like you said :( @notgoodusename

did you include the other files? like vector.h antiaim.cpp and .h it isnt only misc.cpp

**Comment by giangero1** [comment]
image

**Comment by giangero1** [comment]
ye how to use blockbot i toggle it but nothing happens when i get close to the person i wanna block , pretty sure i copy pasted all the commits u posted @notgoodusename

**Comment by notgoodusename** [comment]
Get close to a teamate point to their head and toggle blockbot you will ser

File: Osiris/GUI.cpp [hunk 2]
```diff
@@ -951,8 +958,14 @@ void GUI::renderMiscWindow() noexcept
...
         ImGui::Checkbox("Anti AFK kick", &config.misc.antiAfkKick);
+	ImGui::Checkbox("Block bot", &config.misc.blockbot);
         ImGui::Checkbox("Auto strafe", &config.misc.autoStrafe);
+	ImGui::Combo("Auto strafe style", &config.misc.autostrafestyle, "Legit\0Normal\0");
         ImGui::Checkbox("Bunny hop", &config.misc.bunnyHop);
+	ImGui::Checkbox("Use spam", &config.misc.usespam);
+	ImGui::Checkbox("Slowwalk", &config.misc.slowwalk);
+	ImGui::SameLine();
+	hotkey(config.misc.slowwalkkey);
         ImGui::Checkbox("Fast duck", &config.misc.fastDuck);
```
[the remark this answers is not in the document]
Comment (giangero1) [reply Osiris/GUI.cpp]: lmao and when i used it would stop me :P

**Comment by giangero1** [comment]
> Get close to a teamate point to their head and toggle blockbot you will ser

ye i can't do that because they move and until i open the menu and toggle it they are gone xD we need to add a key for sure

**Comment by giangero1** [comment]
tried it on a teammate bot doens't seem to work idk if it has to be real player or not

**Comment by thomas0aotian** [comment]
@giangero1 i can try to add that keybind rn

**Comment by giangero1** [comment]
> @giangero1 i can try to add that keybind rn

awesome but amm i still tried to toggle it while aiming on a teammates bot head and when he moved it didn't follow him :/

**Comment by thomas0aotian** [comment]
i did it but idt its working

**Comment by giangero1** [comment]
> i did it but idt its working

u don't know if it's working ? wut wut

**Comment by thomas0aotian** [comment]
I don't think

**Comment by HyeN2018** [comment]
Block bot isnt working and would be awesome to have a hotkey

**Comment by agshaidun** [comment]
> 
> 
> Block bot isnt working and would be awesome to have a hotkey

Agree

**Comment by littlehurensohn** [comment]
Antiaim (Legit) still flicking in the demo (after matchmaking) 
I can give u the demo, add me on discord: jessie#2444

**Comment by littlehurensohn** [comment]
> Antiaim (Legit) still flicking in the demo (after matchmaking)
> I can give u the demo, add me on discord: jessie#2444

Btw: And when somebody spectating you.

**Comment by notgoodusename** [comment]
im lazy

**Comment by thomas0aotian** [comment]
I add a hotkey but it's still not working

**Comment by Oly-UK** [comment]
yeah blockbot doesnt work

File: Osiris/GUI.cpp [hunk 3]
```diff
@@ -951,9 +958,18 @@ void GUI::renderMiscWindow() noexcept
         hotkey(config.misc.menuKey);
 
         ImGui::Checkbox("Anti AFK kick", &config.misc.antiAfkKick);
+	ImGui::Checkbox("Block bot", &config.misc.blockbot);
```
Comment (thomas0aotian) [inline Osiris/GUI.cpp]:
```suggestion
	ImGui::Text("Blockbot");
      ImGui::SameLine();
      hotkey(config.misc.blockbotkey);           
```
blockbot keybind gui

File: Osiris/Config.cpp [hunk 4]
```diff
@@ -775,7 +778,13 @@ void Config::load(size_t id) noexcept
         if (miscJson.isMember("Menu key")) misc.menuKey = miscJson["Menu key"].asInt();
         if (miscJson.isMember("Anti AFK kick")) misc.antiAfkKick = miscJson["Anti AFK kick"].asBool();
         if (miscJson.isMember("Auto strafe")) misc.autoStrafe = miscJson["Auto strafe"].asBool();
+	if (miscJson.isMember("Auto strafe style")) misc.autostrafestyle = miscJson["Auto strafe style"].asInt();
+	if (miscJson.isMember("Use spam")) misc.usespam = miscJson["Use spam"].asBool();
+	if (miscJson.isMember("Slowwalk")) misc.slowwalk = miscJson["Slowwalk"].asBool();
+	if (miscJson.isMember("Slowwalk key")) misc.slowwalkkey = miscJson["Slowwalk key"].asBool();
+	if (miscJson.isMember("Slowwalk ammount")) misc.slowwalkammount = miscJson["Slowwalk ammount"].asFloat();
         if (miscJson.isMember("Bunny hop")) misc.bunnyHop = miscJson["Bunny hop"].asBool();
+	if (miscJson.isMember("Block bot")) misc.blockbot = miscJson["Block bot"].asBool();
```
Comment (thomas0aotian) [inline Osiris/Config.cpp]:
```suggestion
	 if (miscJson.isMember("Blockbot key")) misc.blockbotkey = miscJson["Blockbot key"].asInt();
```

File: Osiris/Config.cpp [hunk 5]
```diff
@@ -1504,6 +1516,12 @@ void Config::save(size_t id) const noexcept
         miscJson["Menu key"] = misc.menuKey;
         miscJson["Anti AFK kick"] = misc.antiAfkKick;
         miscJson["Auto strafe"] = misc.autoStrafe;
+	miscJson["Auto strafe style"] = misc.autostrafestyle;
+	miscJson["Use spam"] = misc.usespam;
+	miscJson["Block bot"] = misc.blockbot;
```
Comment (thomas0aotian) [inline Osiris/Config.cpp]:
```suggestion
	miscJson["Block bot"] = misc.blockbotkey;
```

File: Osiris/Config.h [hunk 6]
```diff
@@ -206,7 +209,13 @@ class Config {
         int menuKey{ 0x2D }; // VK_INSERT
         bool antiAfkKick{ false };
         bool autoStrafe{ false };
+	int autostrafestyle{ 0 };
         bool bunnyHop{ false };
+	bool usespam{ false };
+	bool blockbot{ false };
```
Comment (thomas0aotian) [inline Osiris/Config.h]:
```suggestion
	int blockbotkey{ 0 };
```

File: Osiris/Hacks/Misc.cpp [hunk 7]
```diff
@@ -10,7 +9,73 @@
 #include "../SDK/GlobalVars.h"
 #include "../SDK/NetworkChannel.h"
 #include "../SDK/WeaponData.h"
+#include "AntiAim.h"
+void Misc::AutoBlocker(UserCmd* cmd) noexcept
+{
+	auto localPlayer = interfaces.entityList->getEntity(interfaces.engine->getLocalPlayer());
+		float bestdist = 250.f;
+		int index = -1;
+		if (config.misc.blockbot) {
```
Comment (thomas0aotian) [inline Osiris/Hacks/Misc.cpp]:
```suggestion
		if (GetAsyncKeyState(config.misc.blockbotkey)) {
```

File: Osiris/Hacks/Misc.cpp [hunk 8]
```diff
@@ -10,7 +9,73 @@
...
+			angles.y -= localPlayer->eyeAngles().y;
+			angles.normalize();
+			angles.y = std::clamp(angles.y, -180.f, 180.f);
+
+			if (angles.y < 0.0f)
+				cmd->sidemove = 450.f;
+			else if (angles.y > 0.0f)
+				cmd->sidemove = -450.f;
+
+		}
```
Comment (thomas0aotian) [inline Osiris/Hacks/Misc.cpp]:
```suggestion
		} else {
```

File: Osiris/Hacks/Misc.cpp [hunk 9]
```diff
@@ -10,7 +9,73 @@
...
+}
```
Comment (thomas0aotian) [inline Osiris/Hacks/Misc.cpp]:
```suggestion
if (!GetAsyncKeyState(config.misc.blockbotkey))
        return;
    }
}
```

File: Osiris/Hooks.cpp [hunk 10]
```diff
@@ -121,6 +122,8 @@ static bool __stdcall createMove(float inputSampleTime, UserCmd* cmd) noexcept
     Reportbot::run();
     Misc::bunnyHop(cmd);
     Misc::autoStrafe(cmd);
+    Misc::slowwalk(cmd);
```
Comment (thomas0aotian) [inline Osiris/Hooks.cpp]:
```suggestion
    Misc::slowwalk(cmd);
    Misc::AutoBlocker(cmd);
```

**Comment by thomas0aotian** [comment]
blockbot working now!

**Comment by thomas0aotian** [comment]
> yeah blockbot doesnt work

It wasn't working because it wasn't being called in hook

**Comment by Oly-UK** [comment]
legend @thomas0aotian thank you mate

**Comment by notgoodusename** [comment]
why hold and not toggle?

**Comment by thomas0aotian** [comment]
@notgoodusename cuz that's what  most of the cheat did

**Comment by hyperxpro** [comment]
@danielkrupinski Mind reviewing this?

**Comment by giangero1** [comment]
@thomas0aotian LOVE U MAN :O <3

**Comment by agshaidun** [comment]
> 
> 
> blockbot working now!

YOU ROCK!!

**Comment by agshaidun** [comment]
AntiAim.cpp(30,42): warning C4244: '=': conversion from 'double' to 'float', possible loss of data
AntiAim.cpp(41,11): warning C5033: 'register' is no longer a supported storage class
AntiAim.cpp(53,10): warning C4244: '=': conversion from 'double' to 'float', possible loss of data
Misc.cpp(28,15): warning C4244: 'initializing': conversion from 'int' to 'float', possible loss of data

Done building project "Osiris.vcxproj".
========== Build: 1 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========

Is this normal ? 

P.s : cheat work perfectly

**Comment by thomas0aotian** [comment]
@agshaidun warning don't mean nothing it's all cool as lomg as u can build it successfully

**Comment by thomas0aotian** [comment]
@notgoodusename bruh wanna pull the commits in?

**Comment by notgoodusename** [comment]
> @notgoodusename bruh wanna pull the commits in?

bruh moment

**Comment by notgoodusename** [comment]
> @notgoodusename bruh wanna pull the commits in?

Test lby idk if it works

**Comment by thomas0aotian** [comment]
@notgoodusename alright

**Comment by thomas0aotian** [comment]
a huge bunch of errors in hooks.cpp

**Comment by thomas0aotian** [comment]
a huge bunch of errors

**Comment by notgoodusename** [comment]
2 line fix dude

**Comment by nothanku** [comment]
hey, could u possibly add an animation fix for the antiaim and aimbot? so u can see it in thirdperson

**Comment by notgoodusename** [comment]
> hey, could u possibly add an animation fix for the antiaim and aimbot? so u can see it in thirdperson

ok

**Comment by Oly-UK** [comment]
ok so the lby works but every now and then it flicks backwards

**Comment by thomas0aotian** [comment]
AA not working, lby breaker works fine.
Legit AA wasn't desyncing, local head is the real head.

**Comment by littlehurensohn** [comment]
Is fixed legitaa? Or still flicking in demo?

**Comment by notgoodusename** [comment]
idk it shouldnt flick test it out and tell me

**Comment by Oly-UK** [comment]
all working great, good job

**Comment by Oly-UK** [comment]
this chams system makes no sense to me though, trying to implement desync chams as you added fake angle

**Comment by notgoodusename** [comment]
fakr is unchoked ticks, real is choked ticks, normal is both unchocked and chocked ticks

**Comment by notgoodusename** [comment]
oly are u up for testing? if so pass your discord tag

**Comment by 3L337-n2** [comment]
@notgoodusename Csgo updated. Would you update the offsets? I don't really understand where should be update...

**Comment by asraskmekaidasid** [comment]
aa indicator?

**Comment by Evilhelder** [comment]
Sorry for the question but ... has anyone implemented all this on osiris? Does anyone have a DLL with these functions?

**Comment by thomas0aotian** [comment]
S1 else answer I'm going braindead on this one

**Comment by rosesware** [comment]
**How about Resolver?**

**Comment by notgoodusename** [comment]
i been trying to geta resolver, so the idea is that if legs are weird, then fix angle, the thing is id it has a good desync or something like aa jitter it completly breaks the resolver so.. another resolver for jitter aa, static aa and a bruteforce resolver would be p

**Comment by Evilhelder** [comment]
when i inject it, it is randomly crashing the game....

**Comment by 3L337-n2** [comment]
@Evihelder change the visual indexes by yourself then it will be fine.

**Comment by Evilhelder** [comment]
> 
> 
> @Evihelder change the visual indexes by yourself then it will be fine.

ok

**Comment by Evilhelder** [comment]
> 
> 
> @Evihelder change the visual indexes by yourself then it will be fine.

Sorry for the question, but could you be more specific?
My English is very bad, and I'm bad with programming ...

**Comment by 3L337-n2** [comment]
> > @Evihelder change the visual indexes by yourself then it will be fine.
> 
> Sorry for the question, but could you be more specific?
> My English is very bad, and I'm bad with programming ...

https://github.com/danielkrupinski/Osiris/commit/b79aec32b9ef199e0d8a0bb62810f0e0a531fe99

**Comment by notgoodusename** [comment]
check the commits on osiris and search for index

**Comment by Evilhelder** [comment]
I decided to compile the DLL in debug mode to see what happens, and it happened at the time of the crash....
Any tips?

image

**Comment by rosesware** [comment]
If nothing works, I recommend you download .ZIP Osiris source again and add all commits and changes we have here.

**Comment by 3L337-n2** [comment]
@notgoodusename Is the max aim shot/aim inaccuracy are the "Damage Hitchance" in other cheats?

**Comment by 3L337-n2** [comment]
@danielkrupinski And what is "killshot"?

**Comment by Evilhelder** [comment]
> 
> 
> If nothing works, I recommend you download .ZIP Osiris source again and add all commits and changes we have here.

I tried to do that, but ended up breaking the code even more.

Does anyone have the DLL working normally? or at least the project folder to compile?

**Comment by Evilhelder** [comment]
> 
> 
> > If nothing works, I recommend you download .ZIP Osiris source again and add all commits and changes we have here.
> 
> I tried to do that, but ended up breaking the code even more.
> 
> Does anyone have the DLL working normally? or at least the project folder to compile?

Forget it, I managed to resolve by replacing the files that were modified by notgoodusename with the original Osiris files.

I will leave the DLL I made here...
[Release.zip](https://github.com/danielkrupinski/Osiris/files/3888736/Release.zip)

**Comment by rosesware** [comment]
**Games crashing when I enter on deathmatch or any kind type of game mode, any idea about it?**

**Comment by 3L337-n2** [comment]
@Evilhelder What did u change? You only need to download his @notgoodusename source and move the master's Entity.h in it. @rosesware

**Comment by 3L337-n2** [comment]
Update visual indexes pls

**Comment by 3L337-n2** [comment]
@notgoodusename Can we have auto-stop and engine prediction? :)

**Comment by gosu821** [comment]
what is good setting for legit aa? can someone tell cuz im scare about the flick in mm

**Comment by Evilhelder** [comment]
> 
> 
> @Evilhelder What did u change? You only need to download his @notgoodusename source and move the master's Entity.h in it. @rosesware See [#739 (comment)](https://github.com/danielkrupinski/Osiris/pull/739#issuecomment-557940736)

But that's exactly what I did. Here is working normally .... O.o

**Comment by notgoodusename** [comment]
i recomend 0 yaw angle, lby and see fake

**Comment by notgoodusename** [comment]
@3L337-n2 i will work on it when school finishes

**Comment by 3L337-n2** [comment]
@notgoodusename Thx bro, I am looking forward to it.

**Comment by gosu821** [comment]
> i recomend 0 yaw angle, lby and see fake

Thx a lot

**Comment by 3L337-n2** [comment]
Me again, the setting of slowwalk key may have some problem, i set it be ctrl and saved it, but when i load the config again, it becomes mouse 1. Also , may it add a random choked packets or remove scope zoom? I'm new to this, so sorry for some many questions and requests.

**Comment by gosu821** [comment]
> i recomend 0 yaw angle, lby and see fake

hey if i choose side move yaw 0, and thirdperson choked, is it ok?

**Comment by gosu821** [comment]
Can anyone help me about setting legit aa i wanna troll someone, but they still hit my head ez :> thx a lot guys

**Comment by 3L337-n2** [comment]
Thx it fixed, and ~~what is the best wallbang hitchance of auto sinpers?~~

**Comment by 3L337-n2** [comment]
@notgoodusename bruh, I think you should know what happened...
```ImGui::SliderFloat("Slowwalk Amount: ", &config.misc.slowwalkammount, 0.0f, 100.0f, "%.2f");```

**Comment by 3L337-n2** [comment]
Should be this?
 ```ImGui::SliderInt("Slowwalk Amount: ", &config.misc.slowwalkammount, 0, 100, "%d");```

**Comment by thomas0aotian** [comment]
@3L337-n2 no its float

**Comment by thomas0aotian** [comment]
Just f8db8df it's a wrong invert mistake i guess

**Comment by 3L337-n2** [comment]
Crashed when joined match, is it a bug of anti-aim?

**Comment by 3L337-n2** [comment]
Also crashed when not enable anti-aim but died

**Comment by gosu821** [comment]
no problem with crash, but  u save config with blockbot key on, i think this maybe crash the game, dont save cfg with blockbot key on

**Comment by gosu821** [comment]
Everything is fine, notgoodusename's source is fine, no crash

**Comment by 3L337-n2** [comment]
Interesting, i'll try it later

**Comment by 3L337-n2** [comment]
@gosu821 I still got crashed smh, I just added https://github.com/danielkrupinski/Osiris/pull/769 and https://github.com/danielkrupinski/Osiris/pull/432 in codes. Somebody help me? I tried debug but the error is ```0xC0000005: Access violation reading location 0x000033B2.```

**Comment by notgoodusename** [comment]
remove the aa nade fix

**Comment by 3L337-n2** [comment]
Hmm so can we have aa nade fix?

**Comment by steel4me** [comment]
@3L337-n2 what do you mean with -> can we have aa nade fix? Again, i wont be rude, this project is for learning how to code and trying to implement own things by your self, tehre is no subscription or anything that allows you to request changes ;) think about that please

**Comment by notgoodusename** [comment]
i am not cheating anymore :) so cya

**Comment by anarbmw** [comment]
notgoodusename, could u send me ur osiris dll with all these features pls?

**Comment by gosu821** [comment]
> i am not cheating anymore :) so cya

ok cya later, u give us all of thing we need, so thx a lot and hf

**Comment by agshaidun** [comment]
Tips : Turn on AutoStrafe : Legit to make the blockbot better .

**Comment by 3L337-n2** [comment]
@ZPDMiller Will you fix the thridpreson viewmode when aa? It's shaking when on the aa.

**Comment by ZPDMiller** [comment]
Notgoodusername had it working when he pushed an update before, then it broke when he changed some things, I can try and use some of his older code and see if it works

**Comment by 3L337-n2** [comment]
@ZPDMiller Thx you, wait for your good new :)

**Comment by 3L337-n2** [comment]
And it looks like a broken silent aim.

**Comment by 3L337-n2** [comment]
@ZPDMiller I fixed it.

**Comment by ZPDMiller** [comment]
ok

**Comment by XenoxUSser** [comment]
> memes

can someone explain me how i add this to osiris with multiple pull requests?

**Comment by anarbmw** [comment]
why daniel isnt merging these features?

**Comment by xAkiraMiura** [comment]
1: bad code quality
2: pasted
3: uses QAngles over Vector (because its a paste)

**Comment by cojo1989** [comment]
> Thank you

Can you add me on discord? xSniper#6766

**Comment by ZPDMiller** [comment]
> > Thank you
> > @ZPDMiller
> > Can you add me on discord? xSniper#6766
Sent

**Comment by gosu821** [comment]
U back

**Comment by 3L337-n2** [comment]
OOPS, not cheating anymore but code more ? 😂

**Comment by 3L337-n2** [comment]
but why deleted slowwalk?

**Comment by notgoodusename** [comment]
> but why deleted slowwalk?

cus i need to re do it, it works but the code is garbage

**Comment by 3L337-n2** [comment]
Oh, also redo the aa? I'm waiting at it :)

**Comment by agshaidun** [comment]
ma mann has backk :D

**Comment by danielkrupinski** [comment]
Please submit separate PR for each feature, that would make reviewing a lot easier. Thanks.

**Pull request closed without merging**